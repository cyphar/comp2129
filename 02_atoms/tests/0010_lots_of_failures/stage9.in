LOAD a a aa
LOAD
LOAD nonexist
PLAYFROM -2
PLAYFROM ENDa
PLAYFROM aaa
PLAYFROm 0
START 2 3 1
PLAYFROM -2
PLAYFROM ENDa
PLAYFROM aaa
PLAYFROM 0
UNDO
PLAYFROM 0
