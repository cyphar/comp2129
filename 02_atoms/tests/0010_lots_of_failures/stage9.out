Too Many Arguments

Missing Argument

Cannot Load Save

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Game Ready
Red's Turn

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Cannot Undo

Invalid Command

